### ISSUE TITLE
Could you please upload the ONNX model ?

### ISSUE THREAD

**Issue by X901** [gnurt2041/MangaOCR#7]
Hi, 
I tried to converted the model to ONNX format but it not work for me

**Comment by KonstantinDjairo** [comment]
++1 for this

**Comment by HighLiuk** [comment]
So what's the answer?

**Comment by X901** [comment]
> So what's the answer?
> 

there 2 person converted successfully  

you can find it here

https://huggingface.co/xingliao/manga-ocr-onnx-full/tree/main/onnx

and here

https://huggingface.co/l0wgear/manga-ocr-2025-onnx

**Comment by HighLiuk** [comment]
Thanks

**Comment by HighLiuk** [comment]
@X901 but actually those models are the ONNX version of https://github.com/kha-white/manga-ocr/, they're not directly inspired by *this* repo.

By the way, did you find a way to run this?

**Comment by X901** [comment]
> @X901 but actually those models are the ONNX version of https://github.com/kha-white/manga-ocr/, they're not directly inspired by *this* repo.
> 
> By the way, did you find a way to run this?

Sorry for that I was confusing with the name, both models use smiler name it's related to manga-ocr not this repo MangaOCR 😅 

I tried to convert current model but with no luck 

BTW depend on readme on this repo manga-ocr make less error than MangaOCR so it still good idea to try it out 👌

**Comment by HighLiuk** [comment]
Yeah I know it very well but I'm looking for a smaller model with nearly
instant mobile inference. This repo's model claims to be 8mb since it is
based on the PaddleOCR architecture.

*L. Di Fazio*

Il gio 29 gen 2026, 18:41 Basel Baragabah ***@***.***> ha
scritto:
